packages/soundfonts/fontloader.mjs

```diff
@@ -9,14 +10,28 @@ import {
 } from '@strudel/webaudio';
 import gm from './gm.mjs';
 
+export const defaultFontloaderConfig = {
+  soundfontUrl: 'https://felixroos.github.io/webaudiofontdata/sound'
+}
+
+export const fontloaderConfigMap = persistentMap('strudel-config-fontloader', defaultFontloaderConfig);
```

Comment:
not sure if a persistentMap is best. wouldn't a simple global variable be enough? it would make sure the default fallback is used in the next session, so there is no hidden state stored in the browser. The fact that an alternative url is used would be visible in the code at all times

Reply:
you're right, I'll change that.